README.md

```diff
@@ -41,6 +41,30 @@ Here, `<MSGPACK_INCLUDE_PATH>` and `<MMTF_INCLUDE_PATH>` are the paths to the
 
 For your more complicated projects, a `CMakeLists.txt` is included for you.
 
+## Installation
+This is a header only project and hence there is no compilation necessary. If you want to use it in your project you can
+download this repo and add `-I/full/path/to/mmtf-cpp/include` to your `g++` or `clang++` command.  
+*REMINDER* This project depends on [msgpack-c](https://github.com/msgpack/msgpack-c), you must also install that to your system
+or add the `msgpack-c` include directory to your g++ command as well.  
+
+You can also perform a system wide installation with `cmake` and `ninja` (or `make`).  This is generally how you would
+install something for a `Docker` image.
```

Comment:
> This is generally how you would install something for a `Docker` image.

Can we remove the Docker line? I somehow don't see how this is relevant since nothing about `ninja install` (or having `cmake` configure it) is Docker specific...

Reply:
I'm definitely okay removing it, but since we had those two people post issues about properly building/installing into containers I figured the keyword 'docker' would help direct them to the right section of the README. If you still think it's useless though, I'll remove it.

Reply:
I don't remember any of them mentioning Docker or containers, but maybe I missed it. I thought they were just building packages for their favorite Linux distribution...